SU-HSDO/suhumsci
# SHS-5531: Bug: Arrows misaligned on carousel on one site

# READY FOR REVIEW

## Summary
This PR fixes arrows misaligned when carousel is on full width region.

## Need Review By (Date)
03/08

## Urgency
medium

## Steps to Test
1. Create a flexible page on https://clas.suhumsci.loc/ or https://hs-traditiona.suhumsci.loc/
2. Add a Banner image(s) with text box on the Full-width Region. Add some slides
3. Confirm the arrow and dots are well aligned as expected
4. Plus, test this also on Colorful (When testing this, saw that the arrows were misaligned (out of the box) in small viewports)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
